Assets/MixedRealityToolkit.Tools/Microsoft.MixedReality.Toolkit.Tools.asmdef

```diff
@@ -3,7 +3,8 @@
     "references": [
         "Microsoft.MixedReality.Toolkit",
         "Microsoft.MixedReality.Toolkit.Editor.Inspectors",
-        "Microsoft.MixedReality.Toolkit.Editor.Utilities"
+        "Microsoft.MixedReality.Toolkit.Editor.Utilities",
+        "Microsoft.MixedReality.Toolkit.Tools.Runtime"
```

Comment:
Any reason you put this in Tools/Runtime (which will add another visual studio project) instead of under Examples/Input/Scenes/InputAxes? I would actually expect this to just be an example of how to display input axis data...To me, tools is a collection of stuff I can use to build my code, take screenshots, analyze dependencies.

Reply:
Mostly some feedback that this is less of an example of the MRTK and more a tool to build new controllers for the MRTK.
This scene doesn't use any MRTK scripts, but calls directly into Unity Input in order to determine proper mappings for each button / thumbstick / etc on a physical controller.
I'm happy to put it wherever people feel is most correct!

Reply:
Oh okay, makes sense.